src/js/components/Box/Box.js

```diff
@@ -116,29 +116,11 @@ class BoxImpl extends Component {
         widthProp={width}
         heightProp={height}
         responsive={responsive}
-        priorTheme={priorTheme}
```

Comment:
im not sure why we need priorTheme. I've tested the dark and light shadows and they work fine. I'm assuming this was here for a reason, but I cannot seem to reproduce a scenario where this would be needed.
<img width="188" alt="screen shot 2018-12-05 at 12 36 30 am" src="https://user-images.githubusercontent.com/1207250/49500664-120ba980-f826-11e8-84a1-5903af14936b.png">